src/videojs-http-streaming.js

```diff
@@ -1079,7 +1079,14 @@ class VhsHandler extends Component {
 
     this.mediaSourceUrl_ = window.URL.createObjectURL(this.playlistController_.mediaSource);
 
-    this.tech_.src(this.mediaSourceUrl_);
+    // If we are playing HLS with MSE in Safari, add source elements for both the blob and manifest URLs.
+    // The latter will enable Airplay playback on receiver devices.
+    if ((videojs.browser.IS_ANY_SAFARI || videojs.browser.IS_IOS) && this.options_.overrideNative && this.options_.sourceType === 'hls') {
+      this.tech_.addSourceElement(this.mediaSourceUrl_);
```

Comment:
Is it possible that `addSourceElement` will not exist on some techs? Do we want to check for it's existence first?

Reply:
MSE is an extension of HTML5 so we should be able to safely assume that all `tech` references are for the HTML5 tech.

Reply:
Though unlikely, doing a check here adds safety in case someone is using a later version of vhs with the current version of videojs.